nuxeo-distribution/nuxeo-nxr-server/src/main/resources/templates/common-base/nuxeo.defaults

```diff
@@ -39,6 +39,9 @@ nuxeo.vcs.fulltext.analyzer.language=english
 nuxeo.vcs.optimizations.acl.enabled=true
 nuxeo.vcs.optimizations.path.enabled=true
 
+# Enable reliable change tokens in VCS/DBS
+nuxeo.changetoken.enabled=true
+
```

Comment:
naive question: why do we want this to be an "admin" param instead of an "application" param?